<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookTable</class>
 <widget class="QWidget" name="BookTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table_main">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_search">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add_filter">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить фильтр</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_history">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>400</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>210</number>
   </attribute>
   <column>
    <property name="text">
     <string>История</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="table_filter">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>110</y>
     <width>211</width>
     <height>271</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>127</number>
   </attribute>
   <column>
    <property name="text">
     <string>№</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>filter</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button_remove_filter">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить фильтр</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_about">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Информация</string>
   </property>
  </widget>
  <zorder>table_history</zorder>
  <zorder>table_main</zorder>
  <zorder>input_search</zorder>
  <zorder>button_add_filter</zorder>
  <zorder>table_filter</zorder>
  <zorder>button_remove_filter</zorder>
  <zorder>button_about</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
